lib/CardDAV/addressBook.js

```diff
@@ -247,11 +247,6 @@ var jsCardDAV_AddressBook = module.exports = jsDAV_Collection.extend(jsCardDAV_i
                 "privilege" : "{DAV:}read",
                 "principal" : this.addressBookInfo.principaluri,
                 "protected" : true
-            },
-            {
-                "privilege" : "{DAV:}write",
-                "principal" : this.addressBookInfo.principaluri,
-                "protected" : true
```

Comment:
@adosoukey Are there any downsides to removing the 'write' access property? I mean, will the consumer still be allowed to edit his/ her addressbook?